roles/swift-storage-1/tasks/juno.yml

```diff
@@ -71,7 +77,7 @@
   tags: swift
 
 - name: chwon srv/node
-  command: chown swift:swift -R /srv/node 
+  command: chown swift:swift -R /srv/node
```

Comment:
I agree that a module could be used here, but I don't think this needs to be changed at this time since this is the code that was tested.  An issue could be opened to deal with any 'command' module usage.